hasura/migrations/default/1705005696536_squashed/up.sql

```diff
@@ -0,0 +1,30 @@
+alter table "public"."app"
+add column "showcase_img_urls" jsonb null;
```

Comment:
not a fan of having this as a JSON, will require different handling than the rest of the images, can we add this later?

Reply:
ok I will make this text [] instead. And we can have the user store the paths but I can just add a validation on the db that the inputs can only be the fixed paths we predetermined